chromium_src/third_party/blink/renderer/core/input/touch.h

```diff
@@ -0,0 +1,48 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_CORE_INPUT_TOUCH_H_
+#define BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_CORE_INPUT_TOUCH_H_
+
+#include "third_party/blink/renderer/core/frame/local_dom_window.h"
+
+#define screenX                                                            \
+  screenX() const {                                                        \
+    auto* local_dom_window = target()->ToLocalDOMWindow();                 \
+    int delta_screenX = local_dom_window                                   \
```

Comment:
same as above, we're duplicating this logic to subtract screenX from screenX_ChromiumImpl. Callers should just be able to call screenX by itself and it should return either the adjusted value or the original value based on the farbling setting for this site (and flag)

Reply:
I guess it's because this is delta X and not X, but we should have some kind of helper for that

Reply:
Good idea. I created a helper function and use it everywhere: in mouse_event, touch_event, pointer_event. This simplified things quite a bit.